Core/src/org/sleuthkit/autopsy/coreutils/PlatformUtil.java

```diff
@@ -157,6 +158,15 @@ public static String getOcrLanguagePacksPath() {
     public static String getObjectDetectionClassifierPath() {
         return getUserDirectory().getAbsolutePath() + File.separator + CLASSIFIERS_SUBDIRECTORY;
     }
+    
+    /**
+     * Get root path where the user's text classifiers are stored.
+     * 
+     * @return Absolute path to the text classifiers root directory. 
+     */
+    public static String getTextClassifierPath() {
```

Comment:
I'm not sure this shoudl be part of the public API....

Reply:
Okay. I moved this class to TextClassifierUtils. I also moved ensureTextClassifierFolderExists() from Installer to there. If ensureTextClassifierFolderExists() really needs to be in the Installer class, then getTextClassifierPath needs to be public.